packages/dds/sequence/src/intervals/sequenceInterval.ts

```diff
@@ -341,7 +340,6 @@ export class SequenceInterval implements ISerializableInterval {
 		seq?: number,
 		op?: ICombiningOp,
 	): PropertySet | undefined {
-		this.initializeProperties();
```

Comment:
this change isn't like the others. i think it is probably good, but we should pull it out. i also wonder if property manager can be readonly.